app/subscriber/src/features/manage-folder/ManageFolder.tsx

```diff
@@ -121,7 +123,22 @@ export const ManageFolder: React.FC = () => {
 
   return (
     <styled.ManageFolder>
-      <PageSection header={`Manage Folder: ${folder?.name}`}>
+      <PageSection
+        header={
+          <Col className="header-col">
+            <Row className="header-row">
+              <h1 className="title">{`Manage Folder: ${folder?.name}`}</h1>
+              <FaX
```

Comment:
I have a PR coming that has an updated `Action` component which supports a `variant` property.  I'll get you to use it instead of this custom solution.  That way we'll be using the same component throughout the app.